disqualified_officer_descriptions.yml

```diff
@@ -0,0 +1,37 @@
+    'investigation-of-company' : "Disqualification after investigation of company"
+    'matters-determining-unfitness-of-directors' : "Matters for determining unfitness of directors"
+    'competition-infringements' : "Disqualification for competition infringements"
+    'undertaking-for-competition' : "Disqualification undertaking for competition"
+    'participation-in-wrongful-trading' : "Participation in wrongful trading"
+    'conviction-of-offence-punishable-on-indictment-or-conviction-on-indictment-or-summary-conviction' : "Disqualification on conviction of offence punishable only on indictment or either on conviction on indictment or on summary conviction"
+    'persistent-default-under-companies-legislation' : "Disqualification for persistent default under companies legislation"
+    'summary-conviction-of-offence' : "Disqualification on summary conviction of offence"
+    'high-court-to-disqualify-unfit-directors-of-insolvent-companies' : "Duty of High Court to disqualify unfit directors of insolvent companies"
+    'competition-disqualification-order' : "Competition Disqualification Order"
+    'order-disqualifying-person-instructing-unfit-director-of-insolvent-company' : "Order disqualifying person instructing unfit director of insolvent company"
+    'order-disqualifying-person-instructing-unfit-director-other-cases' : "Order disqualifying person instructing unfit director: other cases"
+    'certain-convictions-abroad' : "Disqualification for certain convictions abroad"
+    'undertaking-disqualifying-person-instructing-unfit-director-of-insolvent-company' : "Disqualification undertaking disqualifying person instructing unfit director of insolvent company"
+    'undertaking-from-person-instructing-unfit-director-other-cases' : "Disqualification undertaking from person instructing unfit director: other cases"
+    'undertaking-instead-of-order-under-article-11a' : "Disqualification undertaking instead of an order under Article 11A"
+    'undertaking-instead-of-order-under-article-11d' : "Disqualification undertaking instead of an order under Article 11D"
+    'order-disqualifying-a-person-instructing-an-unfit-director-of-an-insolvent-company' : "Order disqualifying a person instructing an unfit director of an insolvent company"
+    'order-disqualifying-a-person-instructing-an-unfit-director' : "Order disqualifying a person instructing an unfit director"
+    'undertaking-disqualifying-a-person-instructing-an-unfit-director-of-an-insolvent-company' : "Disqualification undertaking disqualifying a person instructing an unfit director of an insolvent company"
+    'undertaking-disqualifying-a-person-instructing-an-unfit-director' : "Disqualification undertaking disqualifying a person instructing an unfit director"
+    'undertaking-instead-of-order-under-article-11a' : "Disqualification undertaking instead of an order under Article 11A"
```

Comment:
This seems to be duplicated. Same for `undertaking-instead-of-order-under-article-11d`